doc/tutorials/debug.rst

```diff
@@ -0,0 +1,371 @@
+.. _acrn-debug:
+
+ACRN Debugging Tools
+####################
+
+This document describes how to use ACRN tools to collect log
+and trace data for debugging.
+
+ACRN Console Command
+********************
+
+The ACRN console provides shell commands for a user to check system states
+and environment settings.
+
+Help
+====
+
+"help" provides information about the supported hypervisor shell commands:
+
+`help`_
+   Displays information about supported hypervisor shell commands
+
+`vm_list`_
+   Lists all VMs, displaying VM Name, VM ID, and VM State (ON=running)
```

Comment:
As far as I can see, `vm_list` will *never* return `ON` for the VM State... it is one of `Created`, `Started`, `Paused` or `Unknown` - see https://github.com/projectacrn/acrn-hypervisor/blob/master/hypervisor/debug/shell.c#L551-L564